browser/speedreader/speedreader_tab_helper.h

```diff
@@ -87,6 +88,10 @@ class SpeedreaderTabHelper
   // mojom::SpeedreaderHost:
   void OnShowOriginalPage() override;
 
+  void ChangeTheme(const std::string& theme);
```

Comment:
If we're doing getters and setters then the function names should be prefixed with those to be consistent. Perhaps this can be "SetTheme" and to match the getter we do "GetTheme"